modules/appeals_api/app/swagger/appeals_api/v1/higher_level_reviews_controller_swagger.rb

```diff
@@ -68,6 +68,7 @@ class AppealsApi::V1::HigherLevelReviewsControllerSwagger
 
   swagger_path '/higher_level_reviews' do
     operation :post, tags: HLR_TAG do
+      key :deprecated, true
```

Comment:
What do you think about adding some logic so these only show up as deprecated in dev?

Reply:
Ah! Ok. I'm going to attempt now but may reach out with questions!

Reply:
need more clarification

Reply:
Sorry I should have been clearer... I was talking about only showing these as deprecated in the lower envs.

So if you had something like this at the top of the file:

`LOWER_ENV_DOCS = !Settings.modules_appeals_api.documentation.path_enabled_flag`

then each of these deprecated places could be something like this:

`key :deprecated, true if LOWER_ENV_DOCS`

The const could named whatever you think makes sense.

Reply:
OK. I think I got it. I was completely forgetting the fact that I was working on a v1 controller. Good times.